pkgs/os-specific/linux/targetcli/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, python, fetchFromGitHub }:
+
+python.pkgs.buildPythonApplication rec {
+  baseName = "targetcli";
+  version = "2.1.fb48";
+  name = "${baseName}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.